examples/showcase-config.yaml

```
@@ -20,7 +20,7 @@ data:
          spec:
            containers:
              - name: backstage
-               image: quay.io/janus-idp/backstage-showcase
+               image: quay.io.com/redhat-developer/rhdh
```

```suggestion
               image: quay.io.com/rhdh-community/rhdh
```

Per https://github.com/redhat-developer/rhdh/blob/main/.github/workflows/next-build-image.yaml#L60, this should be the right repo, right?